resources/styles/molecules/tabs/BpTabs.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-    <section :class="`${block}`">
+    <section :class="`${block}`" ref="el">
```

Comment:
I would avoid naming the ref `el`. Mainly because `el` has a specific meaning in Vue already (https://vuejs.org/api/component-instance.html#el), but also because it's not descriptive. Instead, consider something like `tabsSection`.